### PR TITLE
 CSEN-59 VmFindRequest integration tests were implemented 

### PR DESCRIPTION
**Pull request by MedvedevBW**

### CODE REVIEW

**Comment by MedvedevBW** [comment]
@placccebo

File: src/it/scala/com/bwsw/cloudstack/entities/requests/VmFindRequestIntegrationTestSuite.scala [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+class VmFindRequestIntegrationTestSuite extends FlatSpec with TestEntities {
+  it should "retrieve json string if request consist only default parameters" in {
+    val vmFindRequest = new VmFindRequest
+    val response = mapper.deserialize[VirtualMachinesResponse](executor.executeRequest(vmFindRequest.request))
+
+    assert(response.isInstanceOf[VirtualMachinesResponse])
+  }
+
+  it should "throw ApacheCloudStackClientRequestRuntimeException with status code 431" +
+    " if entity with value of id parameter does not exist" in {
```
Comment (placccebo) [inline src/it/scala/com/bwsw/cloudstack/entities/requests/VmFindRequestIntegrationTestSuite.scala]: with a specified value of id parameter

File: src/it/scala/com/bwsw/cloudstack/entities/requests/VmFindRequestIntegrationTestSuite.scala [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  it should "retrieve json string if request consist default parameters and parameter with incorrect key" in {
+    val incorrectParameterKey = UUID.randomUUID().toString
+    val request = new VmFindRequest().request.addParameter("incorrectParameterKey", "value")
+    val response = mapper.deserialize[VirtualMachinesResponse](executor.executeRequest(request))
+
+    assert(response.isInstanceOf[VirtualMachinesResponse])
+  }
+
+  private def tryExecuteRequest(request: Request): Boolean = {
+    val vmId = UUID.randomUUID()
```
Comment (placccebo) [inline src/it/scala/com/bwsw/cloudstack/entities/requests/VmFindRequestIntegrationTestSuite.scala]: remove these lines

**Review by placccebo — CHANGES_REQUESTED** [review]

File: src/it/scala/com/bwsw/cloudstack/entities/requests/VmFindRequestIntegrationTestSuite.scala (hunk 1 above)
Comment (MedvedevBW) [reply src/it/scala/com/bwsw/cloudstack/entities/requests/VmFindRequestIntegrationTestSuite.scala]: ok

File: src/it/scala/com/bwsw/cloudstack/entities/requests/VmFindRequestIntegrationTestSuite.scala (hunk 2 above)
Comment (MedvedevBW) [reply src/it/scala/com/bwsw/cloudstack/entities/requests/VmFindRequestIntegrationTestSuite.scala]: ok

**Review by placccebo — APPROVED** [review]

**Pull request merged**